test/tests.ts

```diff
@@ -97,14 +97,14 @@ describe('Resolvers', () => {
 		const arch = 'i386'
 		const resolvers = defaultResolvers()
 		const name = resolvers[archDockerfileResolverIdx].name
-		const stream = fs.createReadStream('./tests/test-files/ArchitectureDockerfile/archive.tar')
+		const stream = fs.createReadStream(require.resolve('./test-files/ArchitectureDockerfile/archive.tar'))
 
 		const bundle = new Resolve.Bundle(stream, '', arch)
 		return Resolve.resolveBundle(bundle, resolvers)
 			.then((resolved) => {
 				assert(
 					resolved.projectType === name,
-					'Architecutre specific dockerfile resolver not used'
+					'Architecture specific dockerfile resolver not used'
```

Comment:
While you're at it, might as well add a dash there -- `Architecture-specific` is moar correct I think.